### ISSUE TITLE
game freeze and crash

### ISSUE THREAD

**Issue by goodmangammering** [brandon3055/Draconic-Evolution#985]
## Base information
* Minecraft version: 1.12
* Mod version:1.12-2.3.5.269-universial
* Minecraft Forge version:14.23.0.2531
* Mod Pack: (if applicable) DireWolf20 1.12

## Crash report
```
game freezes then it goe to sky color and crashes 

CRASH REPORT HERE

Rendering Block Entity

java.lang.ArrayIndexOutOfBoundsException: 4
	at com.brandon3055.draconicevolution.client.render.tile.RenderTilePlacedItem.render(RenderTilePlacedItem.java:49)
	at com.brandon3055.draconicevolution.client.render.tile.RenderTilePlacedItem.func_192841_a(RenderTilePlacedItem.java:20)
	at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_192854_a(TileEntityRendererDispatcher.java:155)
	at net.minecraft.client.renderer.tileentity.TileEntityRendererDispatcher.func_180546_a(TileEntityRendererDispatcher.java:130)
	at net.minecraft.client.renderer.RenderGlobal.func_180446_a(RenderGlobal.java:705)
	at net.minecraft.client.renderer.EntityRenderer.func_175068_a(EntityRenderer.java:1345)
	at net.minecraft.client.renderer.EntityRenderer.func_78471_a(EntityRenderer.java:1259)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1062)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1117)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:397)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)




```

**Comment by brandon3055** [comment]
I dont know what black magic bullshit is happening on your client/server but that crash is literally 100% impossible. The only way i can see it happening is if some other mod is screwing with the placed item in some horrible way.

I need more information. Is this a server? If so are you running plugins? is this crash repeatable? Anything else you think may be helpful? 
I would also like to see your entire log if possible. Preferably via pastebin.com

**Comment by goodmangammering** [comment]
no, it is single player. which means no plug-ins. its sort of repetable in that it happens quite often but there is no specific thing that does it. it did only start happening once i started using the energy relay and here is the log ---> [ https://pastebin.com/8JBRdd6G ]    hope it helps

**Comment by jinks** [comment]
I have those crashes, too. To an amount that I stopped using DE placed items for the most part.

(I didn't report it before since I use OptiFine, so my log is probably useless.)

What I observed:
* It happens suddenly while I'm doing literally nothing, as in the game is open but I'm currently reading on the other screen or something
* It seems to happen less when I'm moving around. Being in an inventory for more than a few minutes causes it to happen more often
* The reported block location is often 1 or 2 blocks above where any placed item is, especially when the placed item is below a ceiling
